requirements.txt

```diff
@@ -0,0 +1,19 @@
+astroid==1.5.3
```

Comment:
are these enforced in the downstream project that installs the module?

Reply:
This is a requirement of `pylint`. I will clean this up so I only pin `pylint`, `flake8` and `coverage`. If we pin their versions then we don't need to concern ourselves with 'their' dependencies. After all, `proofreader` will only be used during the builds.